scripts/release-notes.py

```diff
@@ -114,6 +114,9 @@ def main(token, repo, num_commits, exclude_dependabot):
     parser.add_argument('--num-commits', type=int,
                         help='Print this number of commits from git log. ' +
                              '(default: number of commits before the previous tag)')
+    parser.add_argument('--point-markdown-char', type=str, default='*',
+                        help='The markdown character to use for each change log point. ' +
+                             '(default: *)')
```

Comment:
tbh this strikes me as overkill. I think the only reason UI changelog uses `-` is because prettier insists on it, without good reason, and it's something we can easily turn off (exclude changelog from prettier).

Reply:
Okay, I've removed it in [5115eb3](https://github.com/jaegertracing/jaeger/pull/4800/commits/5115eb373420d49e86c90eba2172216e975484a3)